modules/heat_conduction/doc/content/source/userobjects/GrayLambertSurfaceRadiation.md

```diff
@@ -0,0 +1,174 @@
+# GrayLambertSurfaceRadiation
+
+## Description
+
+`GrayLambertSurfaceRadiation` implements the exchange of heat by radiation between
+sidesets by the net radiation method.
+The net radiation method is valid if the surfaces are gray, diffuse radiators.
+The purpose of the `GrayLambertSurfaceRadiation` is to provide the means
+to model radiative exchange for coarse-grained model. `GrayLambertSurfaceRadiation`
+does not compute radiative exchange between element surfaces, but it computes radiative
+transfer only in an average sense between sidesets.
+
+The `GrayLambertSurfaceRadiation` allows coupling radiative heat transfer to regions
+where the heat equation is solved. The net heat transfer caused by radiative heat transfer is
+coupled to the temperature field by `GrayLambertNeumannBC`. `GrayLambertSurfaceRadiation` also supports the
+definition of adiabatic and fixed temperature sidesets. The temperature variable, i.e. the
+variable of the heat conduction equation, does not need to be defined on adiabatic and fixed
+temperature boundaries. This is particularly useful in cavities, where temperature is only
```

Comment:
I do not quite understand this part, is the walls enclosing the cavity also the sidesets adjacent to heat domain?

Reply:
I added an explanation with an example. See if it makes sense when I push it.